drivers/cleanup.py

```diff
@@ -1811,7 +1971,11 @@ def _snapshotCoalesce(self, vdi):
             return False
         Util.log("Coalescing parent %s" % tempSnap)
         util.fistpoint.activate("LVHDRT_coaleaf_delay_2", self.uuid)
+        vhdSize = vdi.getSizeVHD()
+        startTime = time.time()
         self._coalesce(tempSnap)
+        endTime = time.time()
+        self.recordStorageSpeed(startTime, endTime, vhdSize)
```

Comment:
We should probably do this calculation in the _coalesce method as that's used by both snapshot and leaf coalesce, both of which are useful to the speed calculation. We should be able to get the size of the thing we're trying to coalesce rather than the VDI as that's the actual amount of data we're going to copy

Reply:
Dang i thought about just after i pushed. Fixed.